Mathlib/Order/Hom/Lattice.lean

```diff
@@ -286,32 +176,8 @@ theorem Codisjoint.map [OrderTop α] [OrderTop β] [TopHomClass F α β] [SupHom
     (f : F) (h : Codisjoint a b) : Codisjoint (f a) (f b) := by
```

Comment:
Should the remainder of this section, covering `BotHomClass`/`TopHomClass`, not also be moved to `Hom.BoundedLattice`?

More generally: which declarations in this file still require the import of `Hom.Bounded`?

Reply:
After tinkering with it a bit the structure became clear; see new PR description.